### DISCUSSION TITLE
Implement TinaCMS 🐩

### DISCUSSION THREAD

**Discussion by sethdaily** [SSWConsulting/SSW.Rules#1289] [Ideas]
The best way to find out where to put our effort is to be active users of the product ourselves.
Plus, we should be finding bugs before the community does.

Solution: Spike to investigate:
Put TinaCMS on SSW Rules Gatsby (current)
or
Migrate SSW Rules to Next.js

**Comment by drwharris** [comment]
Have to eat dogfood.

**Comment by Calinator444** [comment]
I'm inclined to say Rules should be migrated to Next.js, admittedly knowing little about either Next js or Gatsby, especially if Gatsby is on the way out. I think it'll cost more in the long run to migrate it to migrate to Next js later.

**Comment by JackDevAU** [comment]
💯 Next.js!

**Comment by MattParkerDev** [comment]
Definitely migrate to NextJS. (Once we improve TinaCMS to our needs)
